fluent-syntax/src/ftlstream.js

```diff
@@ -5,10 +5,11 @@ import { ParseError } from "./errors";
 import { includes } from "./util";
 
 const INLINE_WS = [" ", "\t"];
+const ANY_WS = [" ", "\t", "\r", "\n"];
```

Comment:
Not sure if you want to do this as part of this patch or not, but `\t` is a regular char now.

Reply:
oh, cool! updated!